### PR TITLE
CHARTS-2180 python example

### PR DESCRIPTION
**Pull request by rueckstiess**

### CODE REVIEW

File: python/server.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        self.send_header('Content-type','text/html')
+        self.end_headers()
+
+    # handle GET command
+    def do_GET(self):
+        # serve embedding signature
+        if self.path.startswith('/api/embeddedchart/'):
+            self.send_headers()
+            # TODO: parse chart ID from path
+            self.wfile.write(bytes(self.get_embedding_url('51efab7a-3b1f-4fa9-82db-8c31b83b3df9'), 'UTF-8'))
```
Comment (tomhollander) [inline python/server.py]: This TODO needs to be done!

File: python/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    </script>
+</head>
+<body>
+    <h1>MongoDB Charts Embedding Example (Python)</h1>
+    <iframe id="embeddedChart" width="800" height="600" frameborder="0"></iframe>
+
+    <script>
+        renderSignedAuthChart('~REPLACE~CHART_ID', 'embeddedChart'); // Replace first parameter with your Embedded Chart ID
+    </script>
+</body>
```
Comment (tomhollander) [inline python/index.html]: Given these samples are likely to be when people first see the embedding errors, I think we should link to the docs page that explains what these mean. I'll do this across all samples once we have the page to link to.

File: python/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+5. Select the *Verified Signature* tab and turn on embedding.
+
+6. Copy the IFRAME embed code from this dialog and keep it handy
+
+7. Get the Embedding Signing Key from the Admin Settings page. If you are not a Charts Admin you
+   will need to request this info from someone with this role.
+
+Running this Sample
+-------------------
+1. Ensure you have a Python 3.x installed. You can confirm with `python --version`.
```
[the remark this answers is not in the document]
Comment (rueckstiess) [reply python/README.md]: I added a sentence to mention `python3`. If you think it's still not clear enough we could also
- link to a page explaining the key differences
- mention a way to make this example work in python 2 with some minor code changes

File: python/server.py (hunk 1 above)
Comment (rueckstiess) [reply python/server.py]: strange, I definitely implemented that. Perhaps I made a mistake squashing the commits and it got lost. Will re-implement :-)

**Comment by rueckstiess** [comment]
New commit parses the chart ID from the URL, and confirmed that by simply replacing the chart ID in the `index.html` file it renders the other chart instead.

**Pull request merged**